Dockerfile

```diff
@@ -47,7 +47,6 @@ FROM ${BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES} AS assets-precompile
 
 ENV GOVUK_APP_DOMAIN="http://localhost:3000" \
     GOVUK_WEBSITE_ROOT="http://localhost:3000" \
-    GIAS_API_ROOT="https://ea-edubase-api-prod.azurewebsites.net" \
```

Comment:
Do we not want to add the new env variables here?

Reply:
Nope, we don't need this var for the build